web/tests/acceptance/authenticated/projects/project-test.ts

```diff
@@ -217,15 +217,9 @@ module("Acceptance | authenticated/projects/project", function (hooks) {
       .containsText(docOwner)
       .hasAttribute("href", "/documents?owners=%5B%22foo%40bar.com%22%5D");
 
-    assert
-      .dom(DOCUMENT_TYPE)
-      .containsText(docType)
-      .hasAttribute("href", "/documents?docType=%5B%22PRD%22%5D");
+    assert.dom(DOCUMENT_TYPE).containsText(docType);
 
-    assert
-      .dom(DOCUMENT_STATUS)
-      .containsText(docStatus)
-      .hasAttribute("href", "/documents?status=%5B%22Approved%22%5D");
+    assert.dom(DOCUMENT_STATUS).containsText(docStatus);
```

Comment:
I removed interactivity from the status and docType tags. The use case for these filters isn't very strong, especially in comparison to docOwner/productArea — themselves debatable as links in this context — so as not to be worth sacrificing the click area.